controller/persistence/base_entity.go

```diff
@@ -44,12 +44,18 @@ const (
 
 	FieldId             = "id"
 	FieldName           = "name"
+	FieldSemantic       = "semantic"
 	FieldRoleAttributes = "roleAttributes"
 	FieldCreatedAt      = "createdAt"
 	FieldUpdatedAt      = "updatedAt"
 	FieldTags           = "tags"
+
+	SemanticAllOf = "AllOf"
+	SemanticAnyOf = "AnyOf"
 )
 
+var validSemantics = []string{SemanticAllOf, SemanticAnyOf}
```

Comment:
Seems weird this is here, but it doesn't hurt anything.

Reply:
It's shared across multiple entity types, so I put it there b/c I couldn't think of a better place.